src/xdocs/checks/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Redirecting...</title>
+  <meta http-equiv="refresh" content="0; URL=https://checkstyle.org/checks.html" />
```

Comment:
=> `<meta http-equiv="refresh" content="0; URL=../checks.html" />`

This has to work from our versioned documentation.
See https://checkstyle.org/#Previous_Version_Documentation